packages/amplify-provider-awscloudformation/src/initialize-env.ts

```diff
@@ -114,4 +116,21 @@ export async function run(context: $TSContext, providerMetadata: $TSMeta) {
   if (hasMigratedResources) {
     stateManager.setCurrentMeta(undefined, amplifyMeta);
   }
+
+  // Build all the cfn files for categories
+
+  const resourcesToBuild: IAmplifyResource[] = [];
+
+  const { allResources } = await context.amplify.getResourceStatus();
+
+  allResources.forEach(resourceCreated => {
+    resourcesToBuild.push({
+      service: resourceCreated.service as string,
+      category: resourceCreated.category as string,
+      resourceName: resourceCreated.resourceName as string,
+    });
+  });
```

Comment:
nit: This simplifies with `.map()`

```suggestion
  const { allResources } = await context.amplify.getResourceStatus();

  const resourcesToBuild: IAmplifyResource[] = allResources.map(resource => ({
    service: resource.service as string,
    category: resource.category as string,
    resourceName: resource.resourceName as string,
  }));
```